### DISCUSSION TITLE
No historical data

### DISCUSSION THREAD

**Discussion by skoczo** [macxq/foxess-ha#126] [Q&A]
Hi. I configured FoxESS integration and can see current data. What about historic one? In energy panel i can see only data from time when integration were configured
![image](https://user-images.githubusercontent.com/531574/221963477-773845ae-46b5-4d0d-95d8-b0ec2ae6c4c3.png)

This is my energy panel configuration
![image](https://user-images.githubusercontent.com/531574/221963667-b547f6ef-2955-4913-abff-6cc04e30aa66.png)

**Comment by macxq** [comment]
this integration is not fetching historical data ->  https://github.com/macxq/foxess-ha/issues/117
